Makefile

```diff
@@ -24,7 +24,8 @@ fmt:
 	@bin/fmt.sh
 
 .PHONY: build
-build:	
+build: kubeconfig
+	@bin/install-prereqs.sh
```

Comment:
@kyessenov The presubmit is also installing prereq first and then bazel build ter/prow/pilot-presubmit.sh#L45-L47 , also in Makefile, we already have the `setup` task, but the `setup` also include `init.sh` which will invoke bazel build.

The reason that I want to add `kubeconfig` to here is because if developer do not run `make setup`, then there maybe some go tool chain lost and cause bazel build failed, comments?